helm-chart/kuberay-operator/values.yaml

```diff
@@ -89,6 +89,10 @@ env:
 # Otherwise, kuberay will use your custom domain
 # - name: CLUSTER_DOMAIN
 #   value: ""
+# If not set or set to false, when running on OpenShift with Ingress creation enabled, kuberay will create OpenShift route
```

Comment:
Do we use this environment variable anywhere?

Reply:
Yes, in determining the cluster type (with all the commits it got dropped and restored now). As OpenShift cluster can support both routes (default) and ingresses, this allows user to bypass route creation and use ingresses on OpenShift